Configuration/NodeTypes.yaml

```diff
@@ -264,6 +264,10 @@
   ui:
     label: CRON.FormBuilder:NodeTypes.Plugin:fields.checkbox.enhanced
   properties:
+    label:
+      defaultValue: 'Agreement'
```

Comment:
why is the label "Agreement"? We are talking about a generic Checkbox here, right? Where does this text appear?

Reply:
@Magarif ?

Reply:
@baschny It is not my work, the commit is from Andre. I think he could not find an appropriate translation from the German word. Should i change it?